src/SkeletonGenerator.ump

```diff
@@ -9,39 +13,45 @@ class SkeletonGenerator {
     
     internal Path pathRelativeTo;
     internal String fileFrom;
+    lazy boolean hasSpecializedSkeleton;
```

Comment:
Oops, I mixed up Umple's definition of `lazy` vs. [`scala`'s](http://stackoverflow.com/questions/3041253/whats-the-hidden-cost-of-scalas-lazy-val). Ignore it!